Mrs_Cake/Models/Product.cs

```diff
@@ -8,10 +8,10 @@ namespace Mrs_Cake.Models
     public class Product
     {
         public long Id { get; set; }
-        public ProductType Types { get; set; }
+        /*public ProductType Types { get; set; }*/
```

Comment:
commented out these lines for test, they will appear in final version with database